src/player/PlayerQuickItem.cpp

```diff
@@ -104,19 +91,38 @@ bool PlayerRenderer::init()
   DwmEnableMMCSS(TRUE);
 #endif
 
-  mpv_opengl_cb_set_update_callback(m_mpvGL, on_update, (void *)this);
+  static mpv_opengl_init_params opengl_init_params = {
+      get_proc_address, // .get_proc_address
+  };
+
+
+  static mpv_render_param params[] = {
+      {MPV_RENDER_PARAM_API_TYPE, (void *)MPV_RENDER_API_TYPE_OPENGL},
+      {MPV_RENDER_PARAM_OPENGL_INIT_PARAMS, &opengl_init_params},
+#if defined(USE_X11EXTRAS)
+      {MPV_RENDER_PARAM_X11_DISPLAY, QX11Info::display()},
+#endif
+      {(mpv_render_param_type)0}
```

Comment:
Use `nullptr` instead of `0`.

Reply:
It's not a pointer?

Reply:
Though I would use `MPV_RENDER_PARAM_INVALID` here.

Reply:
No it's not a pointer, just and end marker for the list.

Reply:
I see another usage of `{ MPV_RENDER_PARAM_INVALID }` in a similar params object. I'll mimic.